charts/scalar-manager/README.md

```diff
@@ -0,0 +1,26 @@
+# scalar-manager
```

Comment:
Can ignore this file.
It's generated by https://github.com/scalar-labs/helm-charts/blob/main/scripts/update-chart-docs.sh